Gemfile.jruby-1.9.lock

```diff
@@ -94,7 +94,7 @@ GEM
     http_parser.rb (0.6.0-java)
     i18n (0.6.9)
     insist (1.0.0)
-    jar-dependencies (0.1.15)
+    jar-dependencies (0.2.1)
```

Comment:
@purbon can you do an upstream PR on `jruby-kafka` see https://github.com/joekiller/jruby-kafka/blob/master/jruby-kafka.gemspec#L21

Since it is declared as a runtime dependencies we will have it anyway.